mission_control/navigator_missions/nav_missions/symbol_mission.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+import txros
+import rospy
+import std_srvs.srv
+from navigator_msgs.msg import DockShape
+import numpy
```

Comment:
It is a standard conversation to `import numpy as np`. This does a couple things, shortens a lot of the code up (since Numpy is used everywhere) and keeps consistency with current design choices.